src/axom/primal/geometry/CurvedPolygon.hpp

```diff
@@ -74,7 +114,7 @@ class CurvedPolygon
   }
 
   /// Constructor from an array of \a nEdges curves
-  CurvedPolygon(BezierCurveType* curves, int nEdges)
+  CurvedPolygon(CurveType* curves, int nEdges)
```

Comment:
Minor: Should we convert this to an `ArrayView<CurveType>`?
(Perhaps in a follow-up PR)